app/src/main/java/com/example/poste/activities/FolderViewActivity.java

```diff
@@ -84,47 +91,56 @@ private void setListeners() {
 
     private void prepVars() {
         emptyText = findViewById(R.id.folderViewEmptyText);
+        emptyText.setVisibility(View.GONE); // Hide empty text until data is loaded
         folderName = findViewById(R.id.folderNameText);
+        folderName.setVisibility(View.INVISIBLE); // Hide folder name until data is loaded
+        // reset selected folder
```

Comment:
Making these invisible by defaults avoids popin and popout of the placeholders; instead, they'll only be made visible if necessary after updating data